FileDiscovery.py

```diff
@@ -0,0 +1,43 @@
+import glob
+import os
+
+
+class FileDiscovery:
+
+    def __init__(self, dir):
+        self.dir = dir
+        self.__extention_working()
+
+    def __extention_working(self):
```

Comment:
name of this methods is unclear
maybe change to something more like 'load_unique_extensions'